### PR TITLE
4 password handling

### PR DESCRIPTION
**Pull request by Marcell-Puskas**
The project directory structure is new for me. Not sure if Hash model is necessary, but seems logical.

### CODE REVIEW

File: Models/Hash.cs
Comment (KercsiBence) [inline Models/Hash.cs:1]: You felt right about the hash model. You should either delete the struct and leave it as a class with properties, or delete the wrapping class. Either way, check the file name as well to match with the class/struct name. I personaly prefer the class model over struct, but both good.

**Review by KercsiBence — DISMISSED** [review]
Please correct the hash model. Also, check the user model and correct it's constructor or remove if not needed now nor in the future.

File: Services/Implementations/UserService.cs [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
 {
     public class UserService : IUserService
     {
         private readonly DatabaseContext _context;
+        private readonly HashService _hashService;
 
         public UserService(DatabaseContext context)
         {
                _context = context;
+               _hashService = new HashService();
```
[the remark this answers is not in the document]
Comment (Marcell-Puskas) [reply Services/Implementations/UserService.cs:15]: valami

[the remark this answers is not in the document]
Comment (Marcell-Puskas) [reply Services/Implementations/UserService.cs:15]: valami2

**Comment by Marcell-Puskas** [comment]
Done.
The hash model is now just a class, and their field are private.
The user service now users dependency injection.

**Review by vadaszidavid — APPROVED** [review]

**Pull request merged**